src/components/cart/proceed-payment/index.tsx

```diff
@@ -0,0 +1,206 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { colors, sizes, fontFamilies, letterSpacing } from "../../../variables";
+import Searchpath from "../../common/Searchpath";
+import data from "../../common/constants.json";
+import Label from "../../common/label";
+import { useForm, Controller } from "react-hook-form";
+import InputField from "../../common/textbox";
+import RadioButton from "./radioButton";
+
+const ProceedPayment = () => {
+  const { register, handleSubmit, control, reset } = useForm();
+  const [cartItems, setCartItems] = useState(data.cartContents.myCart);
+  const [radioButtons, setRadioButtons] = useState({
+    userCode: false,
+    bestOffers: true,
+  });
+  const handleClick = (e: React.MouseEvent<HTMLElement>) => {
+    setCartItems([]);
+  };
+  const handleRadioClick = (e: React.MouseEvent<HTMLInputElement>) => {
+    if (e.currentTarget.id === "userCode" && radioButtons.userCode === true) {
+      setRadioButtons({ userCode: false, bestOffers: false });
+    } else if (e.currentTarget.id === "userCode") {
+      setRadioButtons({ userCode: true, bestOffers: false });
+    } else if (
+      e.currentTarget.id === "bestOffer" &&
+      radioButtons.bestOffers === true
+    ) {
+      setRadioButtons({ userCode: false, bestOffers: false });
+    } else if (e.currentTarget.id === "bestOffer") {
+      setRadioButtons({ userCode: false, bestOffers: true });
+    } else {
+      return;
+    }
+  };
+  return (
+    <PageSection>
+      <Searchpath path={data.searchPath.menu1} className="grey-dark" />
+      <ContentContainer>
+        <ProgressSection></ProgressSection>
+        <CartSection>
+          <StepsSection>
+            <StepTitleLabel content={data.cartTexts.title} />
+            <CookingInstructionTitle>
+              {data.cartData.cookingInstruction}
+            </CookingInstructionTitle>
+            <CookingInstructionInput
+              name={data.cartData.textboxLabel}
+              isPassword={false}
+            />
+          </StepsSection>
+          <PaymentContainer>
+            <RadioTitle>{data.offersRadio.title}</RadioTitle>
```

Comment:
Why no label here ????

Reply:
RadioTitle converted to RadioTitleLabel using Label styled component.